tests/resources/Harbor-Pages/Project-Members.robot

```diff
@@ -21,69 +21,254 @@ ${HARBOR_VERSION}  v1.1.1
     Sleep  2    
-	Click Element  xpath=${project_member_tag_xpath}
-	Sleep  1	
-	Click Element  xpath=//project-detail//clr-dg-row-master[contains(.,"${user}")]//clr-dg-action-overflow
-	Sleep  1	
+    Click Element  xpath=${project_member_tag_xpath}
+    Sleep  1	
+    Click Element  xpath=//project-detail//clr-dg-row-master[contains(.,"${user}")]//clr-dg-action-overflow
+    Sleep  1	
     Click Element  xpath=//project-detail//clr-dg-action-overflow//button[contains(.,"${role}")]
-	Sleep  2
-	Wait Until Page Contains  ${role}
-	
-	
-	
\ No newline at end of file
+    Sleep  2
+    Wait Until Page Contains  ${role}
+    
+User Can Change Role
+     [arguments]  ${username}
+     Page Should Contain Element  xpath=//project-detail//clr-dg-row-master[contains(.,'${username}')]//clr-dg-action-overflow
+User Can Not Change Role
+     [arguments]  ${username}
+     Page Should Contain Element  xpath=//project-detail//clr-dg-row-master[contains(.,'${username}')]//clr-dg-action-overflow[@hidden=""]
+
+Non-admin View Member Account
+    [arguments]  ${times}
+    Xpath Should Match X Times  //project-detail//clr-dg-action-overflow[@hidden=""]  ${times}
+
+User Can Not Add Member
+    Page Should Not Contain Element  xpath=//project-detail//button//clr-icon
+
+Add Guest Member To Project
+    [arguments]  ${member}
+    Click Element  xpath=//project-detail//button//clr-icon
+    Sleep  1
+    Input Text  xpath=//add-member//input[@id="member_name"]  ${member}
+    #select guest
+    Mouse Down  xpath=//project-detail//form//input[@id='checkrads_guest']
+    Mouse Up  xpath=//project-detail//form//input[@id='checkrads_guest']
+    Click Button  xpath=//project-detail//add-member//button[2]
+    Sleep  1
+
+Delete project member
+    [arguments]  ${member}
+    Click Element  xpath=//project-detail//clr-dg-row-master[contains(.,'${member}')]//clr-dg-action-overflow
+    Click Element  xpath=//project-detail//clr-dg-cell//clr-dg-action-overflow//button[contains(.,"Delete")]   
+    Sleep  1
+    Click Element  xpath=//confiramtion-dialog//button[2]
+    Sleep  1
+
+Create Three Different Users
+    [Arguments]  ${d}
+    Create An New User  ${HARBOR_URL}  username=usera${d}  email=usera${d}@vmware.com  realname=usera${d}  newPassword=Test1@34  comment=harbor
+    Logout Harbor
+    Create An New User  ${HARBOR_URL}  username=userb${d}  email=userb${d}@vmware.com  realname=userb${d}  newPassword=Test1@34  comment=harbor
+    Logout Harbor
+    Create An New User  ${HARBOR_URL}  username=userc${d}  email=userc${d}@vmware.com  realname=userc${d}  newPassword=Test1@34  comment=harbor
+    Logout Harbor
+
+User A Create New Project And Push Image To It
+    [Arguments]  ${ip}  ${d}
+    Sign In Harbor  ${HARBOR_URL}  usera${d}  Test1@34
+    #create project
+    Create An New Project  project${d}
+    #verify can not change role
+    Go Into Project  project${d}
+    Switch To Member
+    User Can Not Change Role  usera${d}
```

Comment:
why add this key words here? Does this func is just to push image to the newly created project?